src/attack_flow_builder/src/assets/builder.config.validator.ts

```diff
@@ -39,6 +40,15 @@ class AttackFlowValidator extends DiagramValidator {
         // Validate properties
         for (const [key, value] of node.props.value) {
             this.validateProperty(id, key, value)
+
+            // Additional validation for reference-based (tactic_ref + technique_ref) properties
+            var STIXregex = /^[a-z][a-z0-9-]+[a-z0-9]--[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}|Null$/
```

Comment:
I might just switch to a const since the regex will be static.

Would it be worthwhile to declare this regex as a class-wide const, or should I keep it within the validateNode function?